src/database/migrations/20220302143333-create-user.js

```diff
@@ -1,33 +1,33 @@
-'use strict';
+/* eslint-disable no-unused-vars */
```

Comment:
remove this comment. Find a better solution

Reply:
Done